client/src/Auth/Auth.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from "react";
+import Container from "@material-ui/core/Container";
```

Comment:
You could import all those components in one line by importing from `@material-ui/core`